.gitignore

```diff
@@ -12,6 +12,8 @@ mattermost-webapp.iml
 
 # disable folders generated by Cypress
 e2e/node_modules
-e2e/cypress/screenshots
+e2e/screenshots
 e2e/cypress/videos
 e2e/results
+e2e/mochawesome-report
+e2e/mochawesome-merged.json
```

Comment:
Let's clean this up with the only folders to ignore.

Reply:
@saturninoabril Pushed up changes to remove those unneeded ignores. Added `.eslintignore` file to handle ignoring e2e/results in linting.